packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1785,6 +1797,23 @@ class StackViewport extends Viewport implements IStackViewport {
     this.initialVOIRange = voiRange;
     this.setProperties({ voiRange });
 
+    // At the moment it appears that vtkImageSlice actors do not automatically
+    // have an RGB Transfer Function created, so we need to create one
+    const cfun = vtkColorTransferFunction.newInstance();
+    let lower = 0;
+    let upper = 1024;
```

Comment:
Can we put a comment where this magic number comes from, even though the answer is "this is the default vtk.js would have made"?